### PR TITLE
feature: BroIDS integration

### PR DESCRIPTION
**Pull request by theopolis**
This adds integrations with BroIDS as an optional feature.

A new and simple Wiki page is added with instructions.

```
./tools/provision.sh install osquery/osquery-local/caf
./tools/provision.sh install osquery/osquery-local/broker
```

```
SKIP_BRO=0 make -j 4
```

Consider this an optional and developer-only feature. We will define some Jenkins tests that periodically build and test this integration. 

@iBigQ is the feature owner and expert. Please direct questions to him.

### CODE REVIEW

**Comment by osqueryer** [comment]
:-1: The commit b5c93e0272e240b911beb4793859883a129c3f46 (Job results: [3387](https://jenkins.osquery.io/job/osqueryPullRequestWindows/3387/)) failed one or more tests (**Windows**).

**Comment by osqueryer** [comment]
:-1: The commit b5c93e0272e240b911beb4793859883a129c3f46 (Job results: [6506](https://jenkins.osquery.io/job/osqueryPullRequestLinux/6506/)) failed one or more tests (**Linux**).

File: docs/wiki/development/broids-support.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+```
+$ ./tools/provision.sh install osquery/osquery-local/caf
+$ ./tools/provision.sh install osquery/osquery-local/broker
+```
+
+## Enable Bro
+
+```
+$ SKIP_BRO=0 make -j 4
```
Comment (iBigQ) [inline docs/wiki/development/broids-support.md]: You have to do a 
```
SKIP_BRO=False make -j 4
```

**Comment by theopolis** [comment]
Hi, I'm going to close this commit, sorry that we never found time to get this merged. We are still open to exploring what deeper looks like but we should re-evaluate based on the current codebase and how dependencies are managed (our new strategy).

**Pull request closed without merging**